### DISCUSSION TITLE
Is there a tailwind utility that can be applied to an input element to make its width fit its content precisely ?

### DISCUSSION THREAD

**Discussion by 1finedev** [tailwindlabs/tailwindcss#6937] [Help]
is there a way to use tailwindcss to allow an input element grow in width according to its inner contents?

**Comment by SamAlhabash** [comment]
I think what you are looking for is the `w-fit` property which is equivalent to `width: fit-content`. 

Here are a coupe of links that can help you solve your problem: 
[width docs on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/width#values)
[width docs on TailwindCSS](https://tailwindcss.com/docs/width)
